doc/drawingElement.md

```diff
@@ -0,0 +1,1186 @@
+
+# Elements to setup canvas (drawing space)
+
+## `pl-drawing` element
+
+A `pl-drawing` element displays a canvas, which can be used to display initial drawing objects or expect drawing objects for grading.
+
+The system of coordinates of the canvas is located at the top/left corner.
```

Comment:
Could we add an image to illustrate the coordinate system?

Reply:
added the figure